src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxVariantDetector.java

```diff
@@ -879,6 +852,153 @@ private static boolean alignmentIsDisjointFromAlphaOmega(final SimpleInterval al
+
+            final CpxVariantType cpxVariant = new CpxVariantType(affectedRefRegion, typeSpecificExtraAttributes());
+            final SimpleInterval pos = new SimpleInterval(affectedRefRegion.getContig(), affectedRefRegion.getStart(), affectedRefRegion.getStart());
+
+            final VariantContextBuilder vcBuilder = new VariantContextBuilder()
+                    .chr(affectedRefRegion.getContig()).start(affectedRefRegion.getStart()).stop(affectedRefRegion.getEnd())
+                    .alleles(AnnotatedVariantProducer.produceAlleles(pos, reference, cpxVariant))
+                    .id(cpxVariant.getInternalVariantId())
+                    .attribute(SVTYPE, cpxVariant.toString())
+                    .attribute(VCFConstants.END_KEY, affectedRefRegion.getEnd())
+                    .attribute(SVLEN, cpxVariant.getSVLength())
+                    .attribute(SEQ_ALT_HAPLOTYPE, new String(altSeq));
+
+            cpxVariant.getTypeSpecificAttributes().forEach(vcBuilder::attribute);
+            return vcBuilder.make();
+        }
+    }
+
+    // evidence used for producing the novel adjacency
+    private static Map<String, String> standardAttributes(final Iterable<AnnotatedContig> annotatedContigs) {
+        final List<AnnotatedContig> sorted = Utils.stream(annotatedContigs)
+                .sorted(Comparator.comparing(tig -> tig.tigWithInsMappings.getSourceContig().contigName))
+                .collect(Collectors.toList());
+
+        final Map<String, String> attributeMap = new HashMap<>();
+        attributeMap.put(GATKSVVCFConstants.TOTAL_MAPPINGS,    String.valueOf(sorted.size()));
+        attributeMap.put(GATKSVVCFConstants.CONTIG_NAMES,      sorted.stream().map(annotatedContig -> annotatedContig.tigWithInsMappings.getSourceContig().contigName).collect(Collectors.joining(VCFConstants.INFO_FIELD_ARRAY_SEPARATOR)));
+
+        // TODO: 2/1/18 question: show we introduce new annotations for them?
+//        attributeMap.put(GATKSVVCFConstants.HQ_MAPPINGS,       tigWithInsMappings.getSourceContig().alignmentIntervals.stream().mapToInt(ai -> ai.mapQual).anyMatch( mq -> mq < CHIMERIC_ALIGNMENTS_HIGHMQ_THRESHOLD ) ? "0" : "1");
+//        attributeMap.put(GATKSVVCFConstants.MAPPING_QUALITIES, tigWithInsMappings.getSourceContig().alignmentIntervals.stream().map(ai -> String.valueOf(ai.mapQual)).collect(Collectors.joining(VCFConstants.INFO_FIELD_ARRAY_SEPARATOR)));
+//        attributeMap.put(GATKSVVCFConstants.ALIGN_LENGTHS,     tigWithInsMappings.getSourceContig().alignmentIntervals.stream().map(ai -> String.valueOf(ai.getSizeOnRead())).collect(Collectors.joining(VCFConstants.INFO_FIELD_ARRAY_SEPARATOR)));
+//        attributeMap.put(GATKSVVCFConstants.MAX_ALIGN_LENGTH,  String.valueOf(tigWithInsMappings.getSourceContig().alignmentIntervals.stream().mapToInt(AlignmentInterval::getSizeOnRead).max().orElse(0)));
+
+        // TODO: 12/11/17 integrate these with those that survived the alignment filtering step?
```

Comment:
I'm a little confused about what would go into this annotation for a complex variant. Don't you already cover this with your segments representation? Maybe you could write a unit test with some examples?